### PR TITLE
Rename databases (fixes #573)

### LINKED ISSUE #573: Database naming: communityregistrationrequests & nations
Both of these databases could store communities or nations depending on whether it is Earth or a nation.  I think it would make more sense to name them as follows:

- communityregistrationrequests -> registrationRequests
- nations -> childPlanets

### PR DESCRIPTION
**Pull request by YangLou1**

### CODE REVIEW

File: couchdb-setup.sh [hunk 1]
```diff
@@ -92,8 +92,8 @@ curl -X PUT $COUCHURL/_global_changes
 curl -X PUT $COUCHURL/meetups
 curl -X PUT $COUCHURL/resources
 curl -X PUT $COUCHURL/courses
-curl -X PUT $COUCHURL/nations
-curl -X PUT $COUCHURL/communityregistrationrequests
+curl -X PUT $COUCHURL/childPlanets
+curl -X PUT $COUCHURL/registrationRequests
```
Comment (lmmrssa) [inline couchdb-setup.sh]: could we keep the name convention and make it 'snake_case'

Comment (paulbert) [reply couchdb-setup.sh]: Ok, let's go with `child_planets` and `registration_requests`

**Comment by paulbert** [comment]
Rebasing to keep this up to date.

This is on hold because I haven't had the time to set up local center -> nation -> community for testing (which I would like to do to make sure this doesn't break any of the connections).  The changes look like they should work, though!

**Comment by paulbert** [comment]
Closing.  Now that we've deployed with the other naming I don't think we should change this.

**Pull request closed without merging**